src/model/format/reminder-base.ts

```diff
@@ -7,13 +7,20 @@ import { Todo } from "./markdown";
 export type ReminderEdit = {
     time?: DateTime,
     rawTime?: string,
-    checked?: boolean
+    status?: ReminderStatus,
 }
 export type ReminderInsertion = {
     insertedLine: string,
     caretPosition: number,
 }
 
+export enum ReminderStatus {
+    Todo = " ",
+    Done = "x",
+    InProgress = "/",
+    Cancelled = "-",
```

Comment:
[Review note]
It seems that these constant based on Tasks Plugin.
https://publish.obsidian.md/tasks/Getting+Started/Statuses/Status+Types